### PR TITLE
Update pullv4data.sh

### PR DESCRIPTION
**Pull request by rh-wadhwani**
Update the V4 pull script to get MOA Information.

### CODE REVIEW

File: V3V4_Cluster_Info/pullv4data.sh [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
-ocm cluster list --managed --padding 100 --columns  id,name,external_id,state,openshift_version,creation_timestamp,activity_timestamp,nodes.master,nodes.infra,nodes.compute,metrics.cpu.total.value,metrics.memory.total.value,cloud_provider.id,region.id,managed,product.id,multi_az,byoc,cluster_admin_enabled,nodes.total,api.url |  awk '{print $1","$2","$3","$4","$5","$6","$7","$8","$9","$10","$11","$12","$13","$14","$15","$16","$17","$18","$19","$20","$21,$22}' > /tmp/ocm_osd.csv
+ocm list clusters --managed --padding 100 --columns  id,name,external_id,state,openshift_version,creation_timestamp,activity_timestamp,nodes.master,nodes.infra,nodes.compute,metrics.cpu.total.value,metrics.memory.total.value,cloud_provider.id,region.id,managed,product.id,multi_az,byoc,cluster_admin_enabled,nodes.total,api.url,properties.moa_cli_version|  awk '{print $1","$2","$3","$4","$5","$6","$7","$8","$9","$10","$11","$12","$13","$14","$15","$16","$17","$18","$19","$20","$21","$22","$23}' > /tmp/ocm_osd.csv
```
Comment (2uasimojo) [inline V3V4_Cluster_Info/pullv4data.sh:2]: Have you considered replacing this `awk` with something simpler, like `sed 's/  */,/g'`?

Comment (rh-wadhwani) [reply V3V4_Cluster_Info/pullv4data.sh:2]: I have not tried sed but awk works like a charm with padding outputs.

Comment (2uasimojo) [reply V3V4_Cluster_Info/pullv4data.sh:2]: My point is that using the above `sed` line you don't need to enumerate every field.

Comment (rh-wadhwani) [reply V3V4_Cluster_Info/pullv4data.sh:2]: It did not work for me. The output was a , after every character. 
Example ",I,D,N,A,M,E,E,X,T,E,R,N,A,L,_,I,D,S,T,A,T,E,O,P,E,N,S,H,I,F,T,"

**Pull request closed without merging**